src/App.vue

```diff
@@ -40,7 +41,7 @@
         ];
       },
       rightWidgets() {
```

Comment:
should be on left-hand side as it's a navigation widget not an information or appearance widget

Reply:
Ok, I will move it across.